### PR TITLE
TIM-4466: AMP analytics - discovery - CNT - custom extension and analytics events PoC

### PR DESCRIPTION
**Pull request by mandelbroo**

### CODE REVIEW

File: extensions/amp-trinity-test/0.1/amp-trinity-test.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      if (this.state === STATE.PLAYING) {
+        trinityProgress += 1;
+        counterEl.innerText = trinityProgress;
+
+        if (trinityProgress === AD_START_TIME) {
+          this.triggerCustomEvent(
+            dict({
+              'eventName': EVENT.AD_STARTED,
+              'state': STATE.PLAYING,
+              'progress': trinityProgress,
```
Comment (industral) [inline extensions/amp-trinity-test/0.1/amp-trinity-test.js:86]: can you send event data we send via postMessage? like deep object enclosure

Comment (mandelbroo) [reply extensions/amp-trinity-test/0.1/amp-trinity-test.js:86]: checking

Comment (mandelbroo) [reply extensions/amp-trinity-test/0.1/amp-trinity-test.js:86]: checking

Comment (mandelbroo) [reply extensions/amp-trinity-test/0.1/amp-trinity-test.js:86]: amp-analytics does not support nested props :(
image

But amp-bind supports
image

Comment (mandelbroo) [reply extensions/amp-trinity-test/0.1/amp-trinity-test.js:86]: I think we can adapt our post messages format before firing to analytics

**Pull request closed without merging**